index.js

```diff
@@ -125,8 +125,15 @@ plugin.manifest = function (pth, opts) {
 			return;
 		}
 
-		firstFile = firstFile || file;
-		manifest[relPath(firstFile.revOrigBase, file.revOrigPath)] = relPath(firstFile.base, file.path);
+		firstFileBase = firstFileBase || file.base;
+
+		var revisionedFile = relPath(firstFileBase, file.path),
+			originalFile = path.join(
```

Comment:
@kevva can you please clarify a bit? I believe I am using multiline var on L130-131.